code/modules/mob/living/living_defense.dm

```diff
@@ -353,6 +354,7 @@
 		B.UpdateIcon()
 
 		B.SetName(A.UpdateName())
+		B.SetName(A.UpdateDesc())
```

Comment:
why's it setting name to desc immediately after setting name?

Reply:
Woops